src/components/presentations/Article/ArticlePage.jsx

```diff
@@ -123,28 +164,36 @@ class ArticlePage extends Component {
                             reportArticle={reportArticle}
                           />
                         </div>
-                      ) : (
-                        <p>Loading...</p>
-                      )}
+                      ) : null}
                     </div>
                   </Grid.Column>
                   <h3>Comments</h3>
-                  {Object.keys(article).length && (
+                  {Object.keys(article).length ? (
                     <InputComment
-                      imageUrl={profile && profile.image_url}
+                      imageUrl={article && article.author.image_url}
```

Comment:
@McHardex Just to clarify, is the input field supposed to have the article authors image or the user that wants to comment?

Reply:
You are commenting on the author's article. So I think the author's image is the one that should be on the initial input. When a user drops his comment, the comment then has his image